src/main/java/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -505,7 +510,7 @@ public int read(byte[] buffer, int offset, int count) throws IOException {
             return result;
         }
 
-        //VisibleForTesting
+        // VisibleForTesting
```

Comment:
I'd love to do this, but that would require a new dependency. That being said, android-support-annotations is probably reasonably well enough isolated from the rest of the support library that it could be reasonable, especially if the dependency didn't propagate to clients (e.g. compile time but not runtime).